deploy/helm/sumologic/values.yaml

```diff
@@ -94,6 +94,14 @@ sumologic:
         repository: public.ecr.aws/sumologic/kubernetes-setup
         tag: 3.0.0
         pullPolicy: IfNotPresent
+      ## Optionally specify an array of pullSecrets.
+      ## They will be added to serviceaccount that is used for Sumo Logic's
+      ## setup job.
+      ## Secrets must be manually created in the namespace.
+      ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+      ##
```

Comment:
I would be good to add information about this parameter to the table available here: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/README.md

Reply:
Should be fixed now, please check